### PR TITLE
Add gravatar helper

### PR DESCRIPTION
**Pull request by tomap**
Copied from hexo

### CODE REVIEW

File: lib/gravatar.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const crypto = require('crypto');
```
Comment (curbengh) [inline lib/gravatar.js]: an opportunity to destructure,
``` js
const { createHash } = require('crypto');
const { stringify } = require('querystring');
```

`crypto.createHash()` -> `createHash()`

File: test/gravatar.spec.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const crypto = require('crypto');
```
Comment (curbengh) [inline test/gravatar.spec.js]: destructure

**Comment by curbengh** [comment]
Rebased. @SukkaW once this is merged, suggest to publish v1.2.0.

File: README.md [hunk 3]
```diff
@@ -102,6 +102,27 @@ full_url_for('/a/path')
 // https://example.com/blog/a/path
 ```
 
+### gravatar(str, options)
+
+Returns the gravatar image url from an email
```
Comment (SukkaW) [inline README.md]: A small typo: missing `.` at the end of the sentence.

**Pull request merged**